addons/sale_project/models/sale_order.py

```diff
@@ -36,6 +38,22 @@ def _compute_is_product_milestone(self):
         for order in self:
             order.is_product_milestone = order.order_line.product_id.filtered(lambda p: p.service_policy == 'delivered_milestones')
 
+    def _compute_show_project_and_task_button(self):
+        is_project_manager = self.env.user.has_group('project.group_project_manager')
+        sols_read_group = self.env['sale.order.line']._read_group([
+            ('order_id', 'in', self.ids),
+            ('order_id.state', 'not in', ['draft', 'sent']),
+            ('is_service', '=', True),
+            '|',
+            ('product_id.invoice_policy', '!=', 'delivery'),
+            ('product_id.service_type', '!=', 'manual'),
```

Comment:
```suggestion
            '|',
                ('product_id.invoice_policy', '!=', 'delivery'),
                ('product_id.service_type', '!=', 'manual'),
```
I like indenting as it makes it clearer which lines the operator applies to. Up to you though :slightly_smiling_face: